configuration-reference/functions/jsonformat.md

```diff
@@ -4,7 +4,7 @@ description: This section contains reference documentation for the JSONFORMAT fu
 
 # JSONFORMAT
 
-Extracts the object value from jsonField based on 'jsonPath', the result type is inferred based on JSON value. This function can only be used in an [ingestion transformation function](../../developers/advanced/ingestion-level-transformations.md).
+Converts a JSON/AVRO complex object to a string. This function can only be used in an [ingestion transformation function](../../developers/advanced/ingestion-level-transformations.md).
```

Comment:
I think it applies to any complex object ingested. Can you please help verify that and probably update both places